keps/sig-node/4800-cpumanager-split-uncorecache/README.md

```diff
@@ -0,0 +1,689 @@
+
+Include CacheID info in podresources API to be able to verify assignment.
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Reference podresources API to determine CPU assignment and CacheID assignment.
+
+###### How can someone using this feature know that it is working for their instance?
+
+Reference podresources API to determine CPU assignment and CacheID assignment.
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [ ] Events
+  - Event Reason: 
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+Measure the time to deploy pods under default settings and compare to the time to deploy pods with align-by-uncorecache enabled. Time difference should be negligible.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- Metrics
+  - Pod deployment time (seconds)
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+N/A
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+
+### Dependencies
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No
+
+
+### Scalability
+
+<!--
+For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them.
+
+For beta, this section is required: reviewers must answer these questions.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### Will enabling / using this feature result in any new API calls?
+
+All of the housekeeping for this feature is node internal, and thus will not require the kubelet request anything new of the apiserver
+
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+NA. 
+
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No
+
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+Pod startup time can directly affected because CPUManager will have to do a few extra steps when scheduling a Pod.
```

Comment:
...but we expect the extra time to be negligible as all the computation is done on RAM

Reply:
I'm going to just add this as additional sentence